eds/blocks/storyteller/storyteller.css

```diff
@@ -51,7 +50,15 @@
   padding-inline-start: 0;
 }
 
-.storyteller .foreground-container a, .storyteller .foreground-container picture, .storyteller p:nth-of-type(1) picture, .storyteller picture.hide-poster {
+.storyteller:not(.content-right) div div:nth-of-type(2) .foreground-container .content-wrapper {
```

Comment:
Should have noticed this earlier -- can we fix the sins of the past and not use directionally oriented class names? Semantically they fail quickly. Should be `content` or `primary-content` or whatever it is.

Reply:
@twhite313 Removed directional oriented class. Replaced with 'primary-content"